src/js/state/migrations/202101201109_moveQueriesStateToGlobal.test.ts

```diff
@@ -0,0 +1,29 @@
+import getTestState from "../../test/helpers/getTestState"
+import migrate from "./202101201109_moveQueriesStateToGlobal"
+import init from "../../../../ppl/queries/initial"
```

Comment:
Since we have to keep these migration files around a long time, I think it's best not to depend on code that we might want to change in the future. If we change the init function, this migration might fail. So I think it's best to copy past the whole expected queries array right into this test case.

Reply:
Oh, good point. Yes I'll change.